althea_kernel_interface/src/traffic_control.rs

```diff
@@ -4,9 +4,39 @@
 //! Rita common in contrast is a simple limitation of the neighbors tunnel, which does not do
 //! any classful categorization. As a result one uses tbf and the other uses the clsssful htb
 
+use ipnetwork::IpNetwork;
+
 use crate::KernelInterface;
 use crate::KernelInterfaceError as Error;
+use std::collections::HashSet;
 use std::net::Ipv4Addr;
+use std::sync::Arc;
+use std::sync::RwLock;
+
+lazy_static! {
+    pub static ref TC_DATASTORE: Arc<RwLock<TcDatastore>> =
+        Arc::new(RwLock::new(TcDatastore::default()));
```

Comment:
I don't like this being in the kernel interface, could we have this stored at the rita-exit level and then passed into the kernel interface functions? Adding state to the kernel interface that's in memory internal in addition to system state seems to really increase complexity to me.

Reply:
Removed all lazy static call from althea kernel interface